drivers/hmis_external_apis/lib/ac_hmis/create_ce_project_configs_20251001.rb

```diff
@@ -0,0 +1,141 @@
+
+      Hmis::Hud::Base.transaction do
+        update_all_configs
+      end
+    end
+
+    private
+
+    def update_all_configs
+      # Most PSH (excluding some) support both waitlist and direct
+      update_configs(
+        Hmis::Hud::Project.hmis.open_on_date.where(project_type: 3).where.not(project_id: PSH_EXCLUDE_IDS),
+        waitlists: true, direct_referrals: true, label: 'PSH (both)', dry_run: dry_run,
+      )
+      # Some PSH support direct only
+      update_configs(
+        Hmis::Hud::Project.hmis.open_on_date.where(project_id: PSH_DIRECT_ONLY_IDS),
+        waitlists: false, direct_referrals: true, label: 'PSH (direct only)', dry_run: dry_run,
+      )
+      # Most RRH (excluding some) support both waitlist and direct
+      update_configs(
+        Hmis::Hud::Project.hmis.open_on_date.where(project_type: 13).where.not(project_id: RRH_EXCLUDE_IDS),
+        waitlists: true, direct_referrals: true, label: 'RRH (both)', dry_run: dry_run,
+      )
+      # Some TH support both waitlist and direct
+      update_configs(
+        Hmis::Hud::Project.hmis.open_on_date.where(project_type: 2).where(project_id: TH_BOTH_IDS),
+        waitlists: true, direct_referrals: true, label: 'TH (both)', dry_run: dry_run,
+      )
+      # Some TH support direct only
+      update_configs(
+        Hmis::Hud::Project.hmis.open_on_date.where(project_type: 2).where(project_id: TH_DIRECT_ONLY_IDS),
+        waitlists: false, direct_referrals: true, label: 'TH (direct only)', dry_run: dry_run,
+      )
+      # Some HP support both waitlist and direct
+      update_configs(
+        Hmis::Hud::Project.hmis.open_on_date.where(project_type: 12).where(project_id: HP_BOTH_IDS),
+        waitlists: true, direct_referrals: true, label: 'HP (both)', dry_run: dry_run,
+      )
+      # Some HP support direct only
+      update_configs(
+        Hmis::Hud::Project.hmis.open_on_date.where(project_type: 12).where(project_id: HP_DIRECT_ONLY_IDS),
+        waitlists: false, direct_referrals: true, label: 'HP (direct only)', dry_run: dry_run,
+      )
+      # Some ES Entry/Exit support direct referrals
+      update_configs(
+        Hmis::Hud::Project.hmis.open_on_date.where(project_type: 0).where(project_id: EE_PROJECT_IDS),
+        waitlists: false, direct_referrals: true, label: 'Entry/Exit', dry_run: dry_run,
+      )
+      # Some Services Only support direct referrals
+      update_configs(
+        Hmis::Hud::Project.hmis.open_on_date.where(project_type: 8).where(project_id: SSO_PROJECT_IDS),
+        waitlists: false, direct_referrals: true, label: 'Services Only', dry_run: dry_run,
+      )
+    end
+
+    def update_configs(scope, waitlists:, direct_referrals:, label: nil, dry_run: false)
+      ce_project = Hmis::Hud::Project.hmis.where(project_name: 'HMIS Coordinated Entry').sole
+      unless waitlists || direct_referrals
+        warn "[#{label}] Skipped: Must support waitlists or direct referrals."
+        return
```

Comment:
should this be an error?

```suggestion
        raise ArguementError, "[#{label}] Skipped: Must support waitlists or direct referrals."
```